### PR TITLE
Nerfs Manlets

### PR DESCRIPTION
**Pull request by MMMiracles**
:cl: MMMiracles
balance: Dwarfism mutation now makes you slightly slower.
/:cl:

[why]: # Tiny legs for tiny man but yet he as fast as big man? Nonsense!

I meant to do this eventually after adding the dwarfism mutation but I forgot to.

### CODE REVIEW

**Comment by LanCartwright** [comment]
smaller size means smaller weight which means easier to propel and move.

I'll only permit this if you benefit giants too.

Manlets have learned.

**Comment by MMMiracles** [comment]
Smaller legs mean shorter strides, meaning less distance traveled compared to big man.

**Comment by Durkel** [comment]
>"beneficial" mutation

>inflicts the worst possible drawback you can have in ss13.

**Comment by GunHog** [comment]
This nerf means that Dwarfism is no longer beneficial.

**Comment by MMMiracles** [comment]
Bypassing tables and being a smaller target in exchange for a slowness that can be negated by one of the various methods to negate slowdown? 

Don't take the mutation if you don't want stumpy legs.

**Comment by LanCartwright** [comment]
Add a benefit to giantism or you don't win my support.

Might as well make welding and flashbangs blind X-ray users or cold resist be unable to reduce burning temperature with your logic.

**Comment by MMMiracles** [comment]
giantism isn't even an actual mutation but a hilarious issue with mob resizing that nobody bothers to fix because its funny.

Dwarfism was meant to have a slowdown but I never got around to actually doing it after the initial PR, so this is me finally bothering to do it. I also don't see an issue with beneficial mutations having their own downsides to accompany their benefits, if anything that sounds more interesting.

bully the manlet-supporters.

**Comment by PKPenguin321** [comment]
Yeah this makes it absolutely non beneficial, I don't think anybody has complained about this enough to warrant destroying it like this

**Comment by Armhulen** [comment]
Dwarfism:
Pros
* climbing tables
* harder to hit
Cons
* inflicts the worst possible drawback you can have in ss13.
"""""""beneficial""""""""

**Comment by ShadowDeath6** [comment]
I like the idea of certain mutations having draws but +1 to speedmod isn't "slight" at all
it's like being legcuffed except you're slightly faster than actual legcuffs and it's permanent until you lose the mutation

**Comment by Repukan** [comment]
Nobody is making you take the mutation though.

If hulk can have drawbacks why can't this?

**Comment by ShadowDeath6** [comment]
too severe a drawback for too negligible a benefit

it gives you slightly increased mobility by making you able to go over tables and being harder to hit in melee but effectively cuts your speed to 75% of what it was, making it harder to dodge around. That's just doing the opposite of what it was intended to be useful for

I think being more prone to damage would be a good change, maybe make the person's brutemod decrease since they're shrimpy and midgets can't take punches well

**Comment by Repukan** [comment]
You can go under tables, which let's you out distance between attackers. Also go under flaps, which makes them go the way around if they have no access. 

Rather than be harder to hit AND yakkety sax around, you can use it to find shortcust and be more strategic on your dodging. 

Furthermore this is a power that comes and is made in the same department that can create flying no slowdown suits or chems to ignore and facilitate faster running speeds. We already shit on mining for perceived free loot and powers, so why do we have this double standard when a department that gets powers with no inherent personal risk and for free (grats you pressed buttons and got lucky) is rebalanced too?

**Comment by FantasticFwoosh** [comment]
Enough with the dwarf hulks that are both unhittable & hard hitting that can cap your knees from under tables in a few punches.

Until now they've had a easy ride, just manlet yourself up and its a large advantage over your opponent, i actually welcome this change and think the rest of the genetics should be revised to some degree.

**Comment by LanCartwright** [comment]
Giantism afaik was a mutation unless I'm mistaken.

Also I've never seen dwarfism being abused to the point to warranting a change. If you're complaining remember that glass tables come death traps for dwarfs.

**Comment by Bawhoppen** [comment]
this does not improve the game

**Comment by Grazyn** [comment]
will they ever learn?

**Comment by FantasticFwoosh** [comment]
@LanCartwright thats the effect of growth hormone from botany, it does not give you any advantages & only distorts your sprite. Giantism (i guess we can technically call hulk giantism) would mean having a larger hitbox or converting to a 64x64 mob change.

**Comment by BeeSting12** [comment]
i thumbs upped it at first but then i realized noone ever complains about manlets and that it looks like a balance change for the sake of changing it

**Comment by FantasticFwoosh** [comment]
Genetics is hard to get into gameplay wise but if you're making yourself a manlet you definitely are doing it so you can antag better, thats the balance reasoning you need.

**Comment by kevinz000** [comment]
>I made myself a fucking dwarf

>I'm still just as durable, powerful, fast, and smart as the other humans, only I can run through places they can't, and my only counter is a glass table

=/

**Comment by Repukan** [comment]
>Genetics
>Gameplay
>Hard

Laughinggreytiders.malf

**Comment by FantasticFwoosh** [comment]
I meant more just going through the process of learning which one is the correct gene and isolating it @Repukan

**Comment by Anonmare** [comment]
@LanCartwright Flashbangs actually do flash x-ray users through walls

**Comment by banthebantz** [comment]
You already nerfed this once.

**Comment by MMMiracles** [comment]
@banthebantz Dwarfism has only gotten buffs if anything since it was added, so no idea what you're talking about.

**Comment by banthebantz** [comment]
@MMMiracles Nevermind I am mistaken about that, still a terrible nerf though

**Comment by ExcessiveUseOfCobblestone** [comment]
Is there a reason for this change?

**Comment by MMMiracles** [comment]
Mostly because dwarfism was intended to have a partial slowdown in the first place when I added it, a little bit because dwarf hulks are like ankle biters that instead of biting your ankles they break your legs.

**Comment by Iamgoofball** [comment]
this is easily the second most powerful mutation in ss13 right now especially when coupled with wavedashing

**Comment by Tacolizard** [comment]
and mutations are the some of the most powerful and easily available buffs.

**Comment by dannno** [comment]
>or you don't win my support.

lmao'd

**Comment by ExcessiveUseOfCobblestone** [comment]
but slowdown is like bad.... like.... REALLY bad.

Make them 0 density and if someone of normal height walks over them they have a X% chance of being stomped on and taking damage. SQUISH THE MANLETS

**Comment by GunHog** [comment]
SPEED IS KING. Reduced speed is a serious nerf, you know!

**Comment by Anonmare** [comment]
Make Dwarfism and Hulk mutually exclusive

Everyone knows manlets can't have GAINS

**Comment by Robustin** [comment]
Jesus christ you guys. Dbl-Esword, Hivemind, and now MANLET being targeted for removal (yes this might as well be removal) in a single week.

This was barely a relevant power in the first place.

**Comment by Repukan** [comment]
When will they learn

**Comment by LanCartwright** [comment]
@Repukan they have. they climb over tables

**Comment by Iamgoofball** [comment]
its that time of the year @Robustin

**Comment by Iamgoofball** [comment]
# have manlets heard of drugs?

**Comment by MMMiracles** [comment]
I think the issue is that the manlets are too short to reach the chem dispenser.

**Comment by Repukan** [comment]
>not being Natty


Never gonna make it

**Comment by KorPhaeron** [comment]
@MMMiracles 

What about a damage multiplier or lower health or something instead?

**Comment by Bawhoppen** [comment]
what about no nerf cause they don't need one?

**Comment by KorPhaeron** [comment]
It passively halves your hitbox and increases your mobility for absolutely 0 downside and would be objectively desireable by any character in any situation, it's lazy and uninteresting

**Comment by Bawhoppen** [comment]
they don't need to be nerfed 
they haven't caused a huge balance problem, and the benefits are far over stated 

(compromise solution: can't shoot guns cause tiny fingers)

**Comment by KorPhaeron** [comment]
Make all but one of their hands occupied by an unremovable offhand slot because their arms are tiny and they need both hands to hold an item a normal person could hold in a single hand

**Comment by Bawhoppen** [comment]
that's fun

**Comment by KorPhaeron** [comment]
Making them be able to only hold one item because their hands are tiny would be funny but I don't know how you'd code that

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -294,6 +295,7 @@ GLOBAL_LIST_EMPTY(mutations_list)
 	owner.update_transform()
 	owner.pass_flags &= ~PASSTABLE
 	owner.visible_message("<span class='danger'>[owner] suddenly grows!</span>", "<span class='notice'>Everything around you seems to shrink..</span>")
+	owner.dna.species.speedmod -= 1
```
Comment (KorPhaeron) [inline code/datums/mutations.dm]: If they gained this then switched species they could get a permanent speedboost

**Comment by Bawhoppen** [comment]
did you delete a comment

File: code/datums/mutations.dm (hunk 1 above)
Comment (MMMiracles) [inline code/datums/mutations.dm]: would initial.speedmod work here or should I look into a different method of nerfing them?

**Comment by kevinz000** [comment]
@Robustin as the king of nerfs you should know how this works.

**Comment by Robustin** [comment]
@KorPhaeron 

>No downside
>Not blowing out manlets by filling the halls with glass tables and watching them instantly crit themselves trying to run around

**Comment by Iamgoofball** [comment]
>build tables in middle of hallway

>get arrested/lynched

**Comment by Iamgoofball** [comment]
remember when the ops meta was "order shotguns and build walls in hallways to maximize spread abuse

**Comment by lzimann** [comment]
@tgstation/commit-access

**Comment by lzimann** [comment]
Speed nerf is not the way to go, plus Kor wants something else too.

File: code/datums/mutations.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/datums/mutations.dm]: If you ever had anything else that modified the species speedmod you could use this to reset it and dodge that penalty as well

**Pull request closed without merging**